tests/foreman/api/test_organization.py

```diff
@@ -304,3 +305,92 @@ def test_negative_update(self, attrs):
         )
         with self.assertRaises(HTTPError):
             response.raise_for_status()
+
+    @skip_if_bug_open('bugzilla', 1103157)
+    def test_bugzilla_1103157(self):
+        """@Test: Create organization and add two compute resources
+        one by one using different transactions and different users
+        to see that they actually added, but not overwrite each other
+
+        @Feature: Organization - Update
+
+        @Steps:
+
+        1. Use the admin user to create an organization and two compute
+        resources. Make one compute resource point at / belong
+        to the organization.
+        2. Create a user and give them the ability to update compute
+        resources and organizations. Have this user make the
+        second compute resource point at / belong to the organization.
+        3. Use the admin user to read information about the organization.
+        Verify that both compute resources are
+        pointing at / belong to the organization.
```

Comment:
reStructuredText syntax requires that the line which starts a list item be indented to the same degree as every line after it.

```
        1. Use the admin user to create an organization and two compute
           resources. Make one compute resource point at / belong to the
           organization.
        2. Create a user and give them the ability to update compute resources
           and organizations. Have this user make the second compute resource
           point at / belong to the organization.
        3. Use the admin user to read information about the organization.
           Verify that both compute resources are pointing at / belong to the
           organization.
```

You can test this out [here](http://rst.ninjs.org/).